Tests/Functional/JMSFunctionalTest.php

```diff
@@ -95,7 +95,7 @@ public function testModelDocumentation()
                 ],
                 'free_form_object' => [
                     'type' => 'object',
-                    'additionalProperties' => true,
```

Comment:
`additionalProperties=true` and `additionalProperties=[]` are equivalent (https://swagger.io/docs/specification/data-models/dictionaries/#free-form)